recipes/libs/fontconfig.yaml

```diff
@@ -0,0 +1,42 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+    PKG_VERSION: "2.13.91"
```

Comment:
2.13.93 is available

Reply:
falling back to 2.13.92, because 2.13.93 doesn't built here.